lib/lotus/commands/generate/abstract.rb

```diff
@@ -0,0 +1,75 @@
+require 'lotus/environment'
+require 'thor'
+
+module Lotus
+  module Commands
+    class Generate < Thor
+      class Abstract
```

Comment:
@brennovich 

Thanks for the feedback!. I'll look into it about the hierarchy. Any suggestion on how you want to share that common behaviour? Also, can you point to a specific location in your PR or give more details? Not sure i understand your "does specifically one thing" reference.